### ISSUE TITLE
Remove the boring warning message by...........

### ISSUE THREAD

**Issue by SeekPoint** [MehdiAB161/Autoencoder-Stability#5]
diff --git a/tools/tools.py b/tools/tools.py
index 9aee363..988e950 100644
--- a/tools/tools.py
+++ b/tools/tools.py
@@ -68,13 +68,13 @@ def log_folder():
 def variable_summaries(var, name):
     with tf.name_scope(name + 'summaries'):
         mean = tf.reduce_mean(var)
-    tf.scalar_summary(name + 'mean', mean)
+    tf.summary.scalar(name + 'mean', mean)
     with tf.name_scope(name + 'stddev'):
         stddev = tf.sqrt(tf.reduce_sum(tf.square(var - mean)))
-    tf.scalar_summary(name + 'sttdev', stddev)
-    tf.scalar_summary(name + 'max', tf.reduce_max(var))
-    tf.scalar_summary(name + 'min', tf.reduce_min(var))
-    tf.histogram_summary(name, var)
+    tf.summary.scalar(name + 'sttdev', stddev)
+    tf.summary.scalar(name + 'max', tf.reduce_max(var))
+    tf.summary.scalar(name + 'min', tf.reduce_min(var))
+    tf.summary.histogram(name, var)

--- a/autoencoder/Train.py
+++ b/autoencoder/Train.py
@@ -81,7 +81,7 @@ class Train(object):
 
     @staticmethod
     def training(loss, optimiser):
-        tf.scalar_summary(loss.op.name, loss)
+        tf.summary.scalar(loss.op.name, loss)
         global_step = tf.Variable(0, name='global_step', trainable=False)
         train_op = optimiser.minimize(loss=loss, global_step=global_step)
         return train_op
